src/BenchmarkDotNet/Loggers/LinqPadLogger.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace BenchmarkDotNet.Loggers
+{
+    public sealed class LinqPadLogger : ILogger
+    {
+        private const string DefaultColor = "";
+
+        // Use Reflection to avoid taking a dependency on the LINQPad assembly
+        static readonly Type linqPadUtil = Type.GetType("LINQPad.Util, LINQPad, Version=1.0.0.0, Culture=neutral, PublicKeyToken=21353812cd2a2db5", throwOnError: true);

```

Comment:
Why do you use `throwOnError: true` here?

Reply:
`Type.GetType(string)` will simply return `null` if the type isn't found, which will immediately cause a `NullReferenceException` in the static constructor. This will throw a more informative `TypeLoadException`.

Reply:
(And I use a named parameter because I personally dislike "naked" Boolean parameter values; this is more self-documenting.)

Reply:
Thus, it will throw an exception once the static field is evaluated. It can be a problem in non-LINQPAD-environment if somebody calls some LinqPadLogger API.
It would be better to have a static instance which is initialized in the static constructor and correctly handle all the exception. This static instance should be initialized only in the LINQPAD environment (IsAvailable=true).

Reply:
Yes, I just realised this.

Reply:
Moved Reflection out of the static constructor and into a `Lazy<bool>` that makes `IsAvailable = true` if all prerequisites are met. 

Reflection will only occur during `DefaultConfig.GetLoggers` if the `AppDomain.CurrentDomain.FriendlyName` test passes, which seems OK to me; let me know if that's not acceptable.